lib/express/sgid.js

```diff
@@ -167,6 +167,40 @@ function config(app, { showLoginPage, serviceProvider }) {
     jwk.use = 'sig'
     res.json({ keys: [jwk] })
   })
+
+  app.get('/.well-known/openid-configuration', async (req, res) => {
+    const issuer = `${req.protocol}://${req.get('host')}`
```

Comment:
personally prefer to rely on server config to tell us what the hostname is, rather than relying on the client request, as per [this comment](https://github.com/datagovsg/sgid-server/pull/382#discussion_r1135017623), but in the absence of an env var which tells us the hostname, this is probably fine